presto-main/src/main/java/com/facebook/presto/sql/planner/plan/TableFinishNode.java

```diff
@@ -55,7 +55,7 @@ public TableFinishNode(
         this.outputs = ImmutableList.copyOf(requireNonNull(outputs, "outputs is null"));
         this.statisticsAggregation = requireNonNull(statisticsAggregation, "statisticsAggregation is null");
         this.statisticsAggregationDescriptor = requireNonNull(statisticsAggregationDescriptor, "statisticsAggregationDescriptor is null");
-        checkArgument(statisticsAggregation.isPresent() == statisticsAggregationDescriptor.isPresent(), "statisticsAggregation and statisticsAggregationDescriptor must be either present or absent");
+        checkArgument(statisticsAggregation.isPresent() == statisticsAggregationDescriptor.isPresent(), "statisticsAggregation and statisticsAggregationDescriptor must both be either present or absent");
```

Comment:
good wording change, but actually belongs to `Collect column statistics on table write: Planner` cmt